### ISSUE TITLE
Reduce Transformer Cannot Work On AWS EC2 Dashboard

### ISSUE THREAD

**Issue by zobnec** [grafana/grafana#25255]
**What happened**:

I config a transform on a chart, and when I click "Select" on Organize fields , nothing happend, then I click Query Panel, then back to Transform Panel, it reports a error called "An unexpected error happened"

**What you expected to happen**:

I expect to rename fields name in the chart

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

Error Detail Info Is:

```
An unexpected error happened
TypeError: Cannot read property 'data' of null
```

**Environment**:
- Grafana version: v7.0.1 (ef5b586d7d)
- Data source type & version: Aws CloudWatch
- OS Grafana is installed on: Ubuntu 18.04 LTS
- User OS & Browser: Chrome
- Grafana plugins: AWS EC CloudWatch
- Others:

**Comment by peterholmberg** [comment]
Thanks for creating an issue @zobnec , can you provide steps to reproduce please.

**Comment by zobnec** [comment]
OK, I will show my operate steps for you @peterholmberg 
1. Edit a pannel in  Aws EC2 dashboard.
<img width="546" alt="屏幕快照 2020-06-02 上午11 47 37" src="https://user-images.githubusercontent.com/7721187/83477901-03f42300-a4c7-11ea-8148-2a06a9bd31cf.png">
2. Then click Transform Tab and click Organize Fields.
<img width="577" alt="屏幕快照 2020-06-02 上午11 49 18" src="https://user-images.githubusercontent.com/7721187/83477986-3bfb6600-a4c7-11ea-87bf-e5d3fd9a70ee.png">
3. I except to add every EC2 instance id a new name, but nothing happened.
4. Then click Query Tab, click back to Transform Tab. Here a error like this:
<img width="525" alt="屏幕快照 2020-06-02 上午11 53 11" src="https://user-images.githubusercontent.com/7721187/83478162-b88e4480-a4c7-11ea-9297-185c37cd767c.png">

**Comment by manuelaznar** [comment]
It happens the same to me too.

I create custom panel

    Grafana version: v7.0.1 (ef5b586)
    Data source type & version: Aws CloudWatch
    OS Grafana is installed on: CentOS 7 
    User OS & Browser: Firefox
    Grafana plugins: AWS EC2 CloudWatch
    Others:

**Comment by daniellee** [comment]
This looks like https://github.com/grafana/grafana/issues/25122 which is fixed in 7.0.3 so looks like a duplicate.
